src/Writer.php

```diff
@@ -20,11 +20,12 @@ class Writer
     /**
      * Convert OpenAPI spec object to JSON data.
      * @param SpecObjectInterface|OpenApi $object the OpenApi object instance.
+     * @param int $flags json_encode() flags
      * @return string JSON string.
      */
-    public static function writeToJson(SpecObjectInterface $object): string
+    public static function writeToJson(SpecObjectInterface $object, int $flags = JSON_PRETTY_PRINT): string
```

Comment:
```suggestion
    public static function writeToJson(SpecObjectInterface $object, int $flags = JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE): string
```

Thanks, `JSON_UNESCAPED_UNICODE` seems to be a useful default together with `JSON_PRETTY_PRINT`.

What do you think?